src/changelog/unreleased.md

```diff
@@ -191,6 +191,8 @@ changelog entry.
 - Removed `KeyEventExtModifierSupplement`, and made the fields `text_with_all_modifiers` and
   `key_without_modifiers` public on `KeyEvent` instead.
 - Move `window::Fullscreen` to `monitor::Fullscreen`.
+- Renamed "super" key to "meta", to match the naming in the W3C specification. The `Super` key is
+  still kept, but deprecated, and only `Meta` will be emitted.
```

Comment:
This is wrong, the old key is not what it was. Users must change. The deprecated warning should also explicitly say so.

Reply:
Updated the text in both places.